### ISSUE TITLE
Please add system requirements to docs / FAQ

### ISSUE THREAD

**Issue by MagicFab** [nuxsmin/sysPass#607]
I couldn't find system requirements information for a new sysPass system.

Perhaps this could be added to the documentation, with a link from the FAQ.

**Comment by nuxsmin** [comment]
Sure, but be aware that sysPass is a lightweight application that should run on almost any system from ten years to now.

Obviously, the bigger db size, the more resources are needed.

BTW, I'll add some advices regarding the optimal requirements and some test results.

Thanks for the feedback!!

**Comment by vanatom** [comment]
I know this is an old request but I think it's a fair question given the growth of cloud-based virtualization and "micro" computing devices like raspberry pi.

I, myself, have deployed sysPass on rPi 3 as well as an EC2 t2.micro instance with great success. However, if I had an idea of the range of memory and disk usage to expect from a range of DB sizes... it would be easier to plan virtualized instances.

Who knows, maybe I could pull it off on an old rPi 1 or even deploy it on a t2.nano instance for like $4/month. :D

**Comment by nuxsmin** [comment]
You're right, some benchmarks would be interesting to work on a resource planning.

Currently sysPass is quite stable (and issues too) so I could take some time to do it.

Thanks for the feedback!

**Comment by vmario89** [comment]
I am playing around with this topic for a while. I was not able to make some good analytical results yet for the public but i came up to make some benchmark/stresstesting by using sysPass API. With the API you can force sysPass to generate significant load by decrypting passwords. 

If you are able to answer a question like this then you are fine: How many users can work in parallel in a real case szenario until the server performance is decreasing the way that frontend will be unusable or server will crash? Some key parameters have to be defined and calculated. For these calculations a list of installed hardware is required too, like: network bandwidth, CPU count, RAM, used web server, library versions, and a lot more stuff. https://github.com/nuxsmin/sysPass/issues/1058
